core/src/main/web/app/helpers/core.js

```diff
@@ -1033,6 +1053,27 @@
             callback(result);
           }
         });
+      },
+      colorize: function (theme) {
+        var colorizedElements = $("body, html");
```

Comment:
Also I feel like this whole function could be a directive on the body. What do you think?